### DISCUSSION TITLE
How to send ctrl-g?

### DISCUSSION THREAD

**Discussion by valankar** [zellij-org/zellij#2667] [Q&A]
Perhaps dumb question, but how do I send ctrl-g to a terminal? ctrl-g enables lock, but pressing it again disables the lock.

**Comment by trekdemo** [comment]
Hey @valankar,

I'm pretty new to zellij, so take my solution with a grain of salt.
I'm coming from tmux and I want to keep my habit of using <kbd>Ctrl</kbd>+<kbd>a</kbd> to unlock zellij.

Hopefully you can adjust my solution to use it with <kbd>Ctrl</kbd>+<kbd>g</kbd>.

Mapping <kbd>Ctrl</kbd>+<kbd>a</kbd> has a major flaw of not being able to jump to the beginning of the current line. So I needed a way to send C-a when I hit <kbd>Ctrl</kbd>+<kbd>a</kbd> twice.

```kdl
keybinds clear-defaults=true {
    locked {
        bind "Ctrl a" { SwitchToMode "normal"; }
    }

    normal {
        // Entering ^A in vim/nvim works like C-v and then C-a
        // Copying it from here did not work for me.
        bind "Ctrl a" { WriteChars "^A"; SwitchToMode "locked"; }
    }
}
```

**Comment by ilyagr** [comment]
See also https://github.com/zellij-org/zellij/issues/2345
